exercise4/isPrime.js

```diff
@@ -0,0 +1,8 @@
+function isPrime(number) {
+  if (number == 2 || number == 1) return true;
+
+  for (let index = 2; index < number; index++) {
+    if (number % index === 0) return false;
+  }
+  return true;
+}
```

Comment:
Hello @devfabien, I love the implementation of the solution, however, this solution returns true when `1` is passed and `1` is not a prime number.  Try modifying this line of code `if (number == 2 || number == 1) return true;`